### PR TITLE
remove urls from logs in Interface

### PR DESCRIPTION
**Pull request by howard-stearns**
https://highfidelity.manuscript.com/f/cases/19754/remove-ALL-logging-of-any-URLs

### CODE REVIEW

**Comment by howard-stearns** [comment]
DNM pending libraries, .qml, and .js.

**Comment by howard-stearns** [comment]
Test plan: smoke test. Then normal acceptance tests following merge.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14815/HighFidelity-Beta-PR14359-05becb8.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14815/HighFidelity-Beta-PR14359-05becb8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14815/HighFidelity-Beta-Interface-PR14359-05becb8.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14824/HighFidelity-Beta-PR14359-e80468e.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14824/HighFidelity-Beta-PR14359-e80468e.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14824/HighFidelity-Beta-Interface-PR14359-e80468e.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14824/HighFidelity-Beta-PR14359-e80468e.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14824/HighFidelity-Beta-Interface-PR14359-e80468e.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14837/HighFidelity-Beta-PR14359-cef7000.apk

**Comment by howard-stearns** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14853/HighFidelity-Beta-PR14359-cef7000.apk

File: interface/resources/qml/hifi/AssetServer.qml [hunk 1]
```diff
@@ -437,15 +435,15 @@ Windows.ScrollingWindow {
...
-                        console.log("Asset Browser - finished uploading: ", fileUrl);
+                        console.log("Asset Browser - finished uploading");
                         reload();
                     } else {
                         uploadSpinner.visible = false;
                         uploadButton.enabled = true;
                         uploadOpen = false;
 
                         if (err !== -1) {
-                            console.log("Asset Browser - error uploading: ", fileUrl, " - error ", err);
```
[the remark this answers is not in the document]
Comment (birarda) [reply interface/resources/qml/hifi/AssetServer.qml]: I think we should keep all of these asset browser error logs. They are useful for the uploading user and for us in debugging when using the asset browser. Won't be exposed to a user who didn't have rights to use the Asset Browser in the first place.

File: scripts/system/libraries/entityList.js [hunk 2]
```diff
@@ -218,7 +218,7 @@ EntityListTool = function(shouldUseEditTabletApp) {
         try {
             data = JSON.parse(data);
         } catch(e) {
-            print("entityList.js: Error parsing JSON: " + e.name + " data " + data);
+            print("entityList.js: Error parsing JSON");
```
[the remark this answers is not in the document]
Comment (huffman) [reply scripts/system/libraries/entityList.js]: This is OK - having these values is only helpful when developing, so having it in the logs for postmortem debugging isn't necessary.

File: scripts/system/html/js/entityProperties.js [hunk 3]
```diff
@@ -1318,15 +1318,6 @@ var selectedEntityProperties;
 var lastEntityID = null;
 var createAppTooltip = new CreateAppTooltip();
 
-function debugPrint(message) {
```
[the remark this answers is not in the document]
Comment (huffman) [reply scripts/system/html/js/entityProperties.js]: This is OK - we don't appear to use this anymore.

File: interface/resources/qml/hifi/AssetServer.qml [hunk 4]
```diff
@@ -81,16 +81,14 @@ Windows.ScrollingWindow {
...
     function doDeleteFile(paths) {
-        console.log("Deleting " + paths);
-
         Assets.deleteMappings(paths, function(err) {
             if (err) {
-                console.log("Asset browser - error deleting paths: ", paths, err);
+                console.log("Asset browser - error deleting paths: ", err);
 
-                box = errorMessageBox("There was an error deleting:\n" + paths + "\n" + err);
+                box = errorMessageBox("There was an error deleting:\n" + err);
```
[the remark this answers is not in the document]
Comment (zfox23) [reply interface/resources/qml/hifi/AssetServer.qml]: ✅

File: interface/resources/qml/hifi/AssetServer.qml [hunk 5]
```diff
@@ -117,11 +115,11 @@ Windows.ScrollingWindow {
 
         Assets.renameMapping(oldPath, newPath, function(err) {
             if (err) {
-                console.log("Asset browser - error renaming: ", oldPath, "=>", newPath, " - error ", err);
+                console.log("Asset browser - error renaming:", err);
```
[the remark this answers is not in the document]
Comment (zfox23) [reply interface/resources/qml/hifi/AssetServer.qml]: ✅

File: interface/resources/qml/hifi/AssetServer.qml (hunk 1 above)
[the remark this answers is not in the document]
Comment (zfox23) [reply interface/resources/qml/hifi/AssetServer.qml]: ✅

File: scripts/system/edit.js [hunk 6]
```diff
@@ -2334,11 +2334,7 @@ var PropertiesTool = function (opts) {
             return;
         }
         var i, properties, dY, diff, newPosition;
-        if (data.type === "print") {
```
[the remark this answers is not in the document]
Comment (huffman) [reply scripts/system/edit.js]: We don't use this anymore, so I'm okay with removing it. It was for a time when we weren't getting logs from WebViews by default

File: interface/resources/qml/hifi/RootHttpRequest.qml [hunk 7]
```diff
@@ -20,7 +20,7 @@ Item {
     // Public function for initiating an http request.
     // REQUIRES parent to be root to have sendToScript!
     function request(options, callback) {
-        console.debug('HttpRequest', JSON.stringify(options));
+        console.debug('HttpRequest');
```
[the remark this answers is not in the document]
Comment (zfox23) [reply interface/resources/qml/hifi/RootHttpRequest.qml]: ✅

File: interface/resources/qml/hifi/RootHttpRequest.qml [hunk 8]
```diff
@@ -33,7 +33,7 @@ Item {
             return;
         }
         delete httpCalls[message.id];
-        console.debug('HttpRequest response', JSON.stringify(message));
+        console.debug('HttpRequest response');
```
[the remark this answers is not in the document]
Comment (zfox23) [reply interface/resources/qml/hifi/RootHttpRequest.qml]: ✅

File: scripts/developer/tests/unit_tests/moduleTests/entity/entityConstructorAPIException.js [hunk 9]
```diff
@@ -2,12 +2,12 @@
 // test module method exception being thrown within main constructor
 (function() {
     var apiMethod = Script.require('../exceptions/exceptionInFunction.js');
-    print(Script.resolvePath(''), "apiMethod", apiMethod);
+    print("apiMethod", apiMethod);
```
[the remark this answers is not in the document]
Comment (huffman) [reply scripts/developer/tests/unit_tests/moduleTests/entity/entityConstructorAPIException.js]: I agree, let's leave these in. I know @thoys has run them in the past when making API updates.

File: interface/resources/qml/hifi/commerce/purchases/Purchases.qml [hunk 10]
```diff
@@ -1009,7 +1009,7 @@ Rectangle {
         try { // Not all calls to onFileOpenChanged() connect an event.
             Window.browseChanged.disconnect(onFileOpenChanged);
         } catch (e) {
-            console.log('Purchases.qml ignoring', e);
+            console.log('Purchases.qml ignoring');
```
[the remark this answers is not in the document]
Comment (zfox23) [reply interface/resources/qml/hifi/commerce/purchases/Purchases.qml]: ✅

File: interface/resources/qml/hifi/dialogs/TabletAssetServer.qml [hunk 11]
```diff
@@ -81,16 +81,16 @@ Rectangle {
...
-        console.log("Deleting " + paths);
+        console.log("Deleting paths");
 
         Assets.deleteMappings(paths, function(err) {
             if (err) {
-                console.log("Asset browser - error deleting paths: ", paths, err);
+                console.log("Asset browser - error deleting paths: ", err);
 
-                box = errorMessageBox("There was an error deleting:\n" + paths + "\n" + err);
+                box = errorMessageBox("There was an error deleting:\n" + err);
```
[the remark this answers is not in the document]
Comment (zfox23) [reply interface/resources/qml/hifi/dialogs/TabletAssetServer.qml]: ✅

File: interface/src/ModelPackager.cpp [hunk 12]
```diff
@@ -83,10 +83,10 @@ bool ModelPackager::loadModel() {
                                  QString("ModelPackager::loadModel()"),
                                  QString("Could not open FST file %1").arg(_modelFile.filePath()),
                                  QMessageBox::Ok);
-            qWarning() << QString("ModelPackager::loadModel(): Could not open FST file %1").arg(_modelFile.filePath());
+            qWarning() << "ModelPackager::loadModel(): Could not open FST file";
```
[the remark this answers is not in the document]
Comment (zfox23) [reply interface/src/ModelPackager.cpp]: ✅

File: interface/src/ModelPackager.cpp [hunk 13]
```diff
@@ -102,19 +102,19 @@ bool ModelPackager::loadModel() {
                              QString("ModelPackager::loadModel()"),
                              QString("Could not open FBX file %1").arg(_fbxInfo.filePath()),
                              QMessageBox::Ok);
-        qWarning() << QString("ModelPackager::loadModel(): Could not open FBX file %1").arg(_fbxInfo.filePath());
+        qWarning() << "ModelPackager::loadModel(): Could not open FBX file";
         return false;
     }
     try {
-        qCDebug(interfaceapp) << "Reading FBX file : " << _fbxInfo.filePath();
+        qCDebug(interfaceapp) << "Reading FBX file";
```
[the remark this answers is not in the document]
Comment (zfox23) [reply interface/src/ModelPackager.cpp]: ✅

File: libraries/script-engine/src/ScriptEngine.cpp [hunk 14]
```diff
@@ -419,10 +418,10 @@ void ScriptEngine::waitTillDoneRunning() {
                 workerThread->quit();
 
                 if (isEvaluating()) {
-                    qCWarning(scriptengine) << "Script Engine has been running too long, aborting:" << getFilename();
+                    qCWarning(scriptengine) << "Script Engine has been running too long, aborting.";
```
[the remark this answers is not in the document]
Comment (huffman) [reply libraries/script-engine/src/ScriptEngine.cpp]: I think we should leave these in. These look like rare, exceptional cases, so shouldn't result in excessive spam.

File: libraries/entities/src/EntityEditFilters.cpp [hunk 15]
```diff
@@ -223,7 +222,7 @@ static bool hasCorrectSyntax(const QScriptProgram& program) {
         const auto error = syntaxCheck.errorMessage();
         const auto line = QString::number(syntaxCheck.errorLineNumber());
         const auto column = QString::number(syntaxCheck.errorColumnNumber());
-        const auto message = QString("[SyntaxError] %1 in %2:%3(%4)").arg(error, program.fileName(), line, column);
+        const auto message = QString("[SyntaxError] %1 in %2(%3)").arg(error, line, column);
```
[the remark this answers is not in the document]
Comment (zfox23) [reply libraries/entities/src/EntityEditFilters.cpp]: ✅

File: libraries/entities/src/EntityEditFilters.cpp [hunk 16]
```diff
@@ -236,8 +235,8 @@ static bool hadUncaughtExceptions(QScriptEngine& engine, const QString& fileName
         const auto line = QString::number(engine.uncaughtExceptionLineNumber());
         engine.clearExceptions();
 
-        static const QString SCRIPT_EXCEPTION_FORMAT = "[UncaughtException] %1 in %2:%3";
-        auto message = QString(SCRIPT_EXCEPTION_FORMAT).arg(exception, fileName, line);
+        static const QString SCRIPT_EXCEPTION_FORMAT = "[UncaughtException] %1 on line %2";
```
[the remark this answers is not in the document]
Comment (zfox23) [reply libraries/entities/src/EntityEditFilters.cpp]: ✅

File: libraries/octree/src/Octree.cpp [hunk 17]
```diff
@@ -699,14 +696,12 @@ bool Octree::readFromFile(const char* fileName) {
...
     QFile file(qFileName);
     if (!file.open(QIODevice::ReadOnly)) {
-        qCritical() << "Cannot open gzipped json file for reading: " << qFileName;
         return false;
     }
     QByteArray compressedJsonData = file.readAll();
     QByteArray jsonData;
 
     if (!gunzip(compressedJsonData, jsonData)) {
-        qCritical() << "json File not in gzip format: " << qFileName;
```
[the remark this answers is not in the document]
Comment (zfox23) [reply libraries/octree/src/Octree.cpp]: ✅

File: libraries/octree/src/OctreeDataUtils.cpp [hunk 18]
```diff
@@ -24,7 +24,6 @@
 bool readOctreeFile(QString path, QJsonDocument* doc) {
     QFile file(path);
     if (!file.open(QIODevice::ReadOnly)) {
-        qCritical() << "Cannot open json file for reading: " << path;
```
[the remark this answers is not in the document]
Comment (zfox23) [reply libraries/octree/src/OctreeDataUtils.cpp]: ✅

File: libraries/script-engine/src/ScriptEngines.cpp [hunk 19]
```diff
@@ -112,7 +112,7 @@ QUrl expandScriptUrl(const QUrl& rawScriptURL) {
 
             QUrl defaultScriptsLoc = PathUtils::defaultScriptsLocation();
             if (!defaultScriptsLoc.isParentOf(url)) {
-                qCWarning(scriptengine) << "Script.include() ignoring file path" << rawScriptURL
+                qCWarning(scriptengine) << "Script.include() ignoring file path"
```
[the remark this answers is not in the document]
Comment (zfox23) [reply libraries/script-engine/src/ScriptEngines.cpp]: `url` in this case refers to a local file, so i don't think that's right.

File: scripts/developer/tests/unit_tests/moduleTests/entity/entityPreloadAPIError.js [hunk 20]
```diff
@@ -2,12 +2,12 @@
 // test module method exception being thrown within preload
 (function() {
     var apiMethod = Script.require('../exceptions/exceptionInFunction.js');
-    print(Script.resolvePath(''), "apiMethod", apiMethod);
+    print("apiMethod", apiMethod);
```
[the remark this answers is not in the document]
Comment (zfox23) [reply scripts/developer/tests/unit_tests/moduleTests/entity/entityPreloadAPIError.js]: with whom?

File: scripts/modules/request.js [hunk 21]
```diff
@@ -75,9 +75,3 @@ module.exports = {
         httpRequest.send(options.body || null);
     }
 };
-
-// ===========================================================================================
-// @function - debug logging
-function debug() {
```
[the remark this answers is not in the document]
Comment (zfox23) [reply scripts/modules/request.js]: ✅

File: scripts/developer/tests/unit_tests/moduleTests/entity/entityConstructorAPIException.js (hunk 9 above)
[the remark this answers is not in the document]
Comment (zfox23) [reply scripts/developer/tests/unit_tests/moduleTests/entity/entityConstructorAPIException.js]: ✅

File: libraries/script-engine/src/ScriptEngine.cpp (hunk 14 above)
[the remark this answers is not in the document]
Comment (zfox23) [reply libraries/script-engine/src/ScriptEngine.cpp]: ✅

**Comment by zfox23** [comment]
i made changes to resolve all comments NOT requiring further clarification by techleads as of this moment (2018-11-08 10:08AM).
i didn't make any changes regarding "the new spec".

File: libraries/networking/src/MappingRequest.cpp [hunk 22]
```diff
@@ -96,9 +96,6 @@ void GetMappingRequest::doStart() {
             // if it did grab that re-directed path
             if (_wasRedirected) {
                 _redirectedPath = message->readString();
-                qDebug() << "Got redirected from " << _path << " to " << _redirectedPath;
```
[the remark this answers is not in the document]
Comment (birarda) [reply libraries/networking/src/MappingRequest.cpp]: We can remove this

File: libraries/networking/src/ResourceCache.cpp [hunk 23]
```diff
@@ -768,21 +763,21 @@ bool Resource::handleFailedRequest(ResourceRequest::Result result) {
     bool willRetry = false;
     switch (result) {
         case ResourceRequest::Result::Timeout: {
-            qCDebug(networking) << "Timed out loading" << _url << "received" << _bytesReceived << "total" << _bytesTotal;
+            qCDebug(networking) << "Timed out, received" << _bytesReceived << "total" << _bytesTotal;
```
[the remark this answers is not in the document]
Comment (birarda) [reply libraries/networking/src/ResourceCache.cpp]: I think we should keep this with the file name portion

File: libraries/networking/src/MappingRequest.cpp (hunk 22 above)
[the remark this answers is not in the document]
Comment (ZappoMan) [reply libraries/networking/src/MappingRequest.cpp]: @birarda - this one? where are these logs at? on the server side?

[the remark this answers is not in the document]
Comment (birarda) [reply libraries/networking/src/MappingRequest.cpp]: This is printed on the client. If we need this debug we can print it at the server.

File: interface/src/assets/ATPAssetMigrator.cpp [hunk 24]
```diff
@@ -209,7 +209,7 @@ void ATPAssetMigrator::migrateResource(ResourceRequest* request) {
     // add this URL to our hash of AssetUpload to original URL
     _originalURLs.insert(upload, request->getUrl());
 
-    qCDebug(asset_migrator) << "Starting upload of asset from" << request->getUrl();
+    qCDebug(asset_migrator) << "Starting upload of asset";
```
[the remark this answers is not in the document]
Comment (zfox23) [reply interface/src/assets/ATPAssetMigrator.cpp]: ✅

File: scripts/developer/tests/unit_tests/moduleTests/entity/entityPreloadAPIError.js (hunk 20 above)
[the remark this answers is not in the document]
Comment (ZappoMan) [reply scripts/developer/tests/unit_tests/moduleTests/entity/entityPreloadAPIError.js]: @zfox23 - see if you can determine the author of the file from gitblame

File: libraries/networking/src/AssetUpload.cpp [hunk 25]
```diff
@@ -76,10 +76,6 @@ void AssetUpload::start() {
     
     // ask the AssetClient to upload the asset and emit the proper signals from the passed callback
     auto assetClient = DependencyManager::get<AssetClient>();
-   
-    if (!_filename.isEmpty()) {
-        qCDebug(asset_client) << "Attempting to upload" << _filename << "to asset-server.";
```
[the remark this answers is not in the document]
Comment (zfox23) [reply libraries/networking/src/AssetUpload.cpp]: ✅

File: libraries/networking/src/ResourceCache.cpp (hunk 23 above)
[the remark this answers is not in the document]
Comment (zfox23) [reply libraries/networking/src/ResourceCache.cpp]: ✅

File: scripts/developer/tests/unit_tests/moduleTests/entity/entityPreloadAPIError.js (hunk 20 above)
[the remark this answers is not in the document]
Comment (zfox23) [reply scripts/developer/tests/unit_tests/moduleTests/entity/entityPreloadAPIError.js]: it's @humbletim

[the remark this answers is not in the document]
Comment (zfox23) [reply scripts/developer/tests/unit_tests/moduleTests/entity/entityPreloadAPIError.js]: i can't imagine this one matters. it'll almost certainly print a local file path every time. i'll put it back in

[the remark this answers is not in the document]
Comment (zfox23) [reply scripts/developer/tests/unit_tests/moduleTests/entity/entityPreloadAPIError.js]: ✅

File: scripts/system/libraries/entityList.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (zfox23) [reply scripts/system/libraries/entityList.js]: ✅

File: scripts/system/html/js/entityProperties.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (zfox23) [reply scripts/system/html/js/entityProperties.js]: ✅

File: scripts/system/edit.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (zfox23) [reply scripts/system/edit.js]: ✅

**Comment by zfox23** [comment]
i made changes to resolve all comments NOT requiring further clarification by techleads as of this moment (2018-11-08 10:41AM).

there is one outstanding comment as of this moment.

i didn't make any changes regarding "the new spec".

File: interface/resources/qml/hifi/AvatarApp.qml [hunk 26]
```diff
@@ -62,7 +62,7 @@ Rectangle {
                     }
                 }
                 catch(err) {
-                    console.error(err);
+                    //console.error(err);
```
[the remark this answers is not in the document]
Comment (zfox23) [reply interface/resources/qml/hifi/AvatarApp.qml]: discussed via slack
✅

File: libraries/script-engine/src/ScriptEngines.cpp (hunk 19 above)
[the remark this answers is not in the document]
Comment (ZappoMan) [reply libraries/script-engine/src/ScriptEngines.cpp]: lets call this good... resolving

**Comment by zfox23** [comment]
as of 11:28am, it looks like all comments are resolved

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14939/HighFidelity-Beta-PR14359-1b2593f.apk

**Comment by howard-stearns** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15003/HighFidelity-Beta-PR14359-1b2593f.apk

**Comment by rickcarr044** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15017/HighFidelity-Beta-PR14359-be5dc36.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15017/HighFidelity-Beta-PR14359-be5dc36.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15017/HighFidelity-Beta-Interface-PR14359-be5dc36.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15017/HighFidelity-Beta-PR14359-be5dc36.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15017/HighFidelity-Beta-Interface-PR14359-be5dc36.exe

**Comment by rickcarr044** [comment]
Stopped testing 14359
Leaving for more investigation on Monday and to open this up for Luis to also investigate.  There are some https mentions in the attached log file.  Some are outside the scope of this ticket.  A couple mention jpgs, or scripts.  For examples: 

source: https://staging.highfidelity.com/assets/places/thumbnail-default-place-e5a3f33e773ab699495774990a562f9f7693dc48ef90d8be6985c645a0280f75.png: failed to load
and
Error: Result of expression '_attachmentData.joint' [undefined] is not an object. in https://hifi-content.s3.amazonaws.com/DomainContent/AvatarStore/attachmentItemScript.js:138
and
Web Entity JS message: https://s.ytimg.com/yts/jsbin/www-widgetapi-vfl8_u944/www-widgetapi.js 93 Error while parsing the 'allow' attribute: 'accelerometer;', 'autoplay;', 'encrypted-media;', 'gyroscope;', 'picture-in-picture' are invalid feature names.

[hifi-log.txt](https://github.com/highfidelity/hifi/files/2567934/hifi-log.txt)

**Comment by ZappoMan** [comment]
> Stopped testing 14359
> Leaving for more investigation on Monday and to open this up for Luis to also investigate. There are some https mentions in the attached log file. Some are outside the scope of this ticket. A couple mention jpgs, or scripts. For examples:
> 
> source: https://staging.highfidelity.com/assets/places/thumbnail-default-place-e5a3f33e773ab699495774990a562f9f7693dc48ef90d8be6985c645a0280f75.png: failed to load
> and
> Error: Result of expression '_attachmentData.joint' [undefined] is not an object. in https://hifi-content.s3.amazonaws.com/DomainContent/AvatarStore/attachmentItemScript.js:138
> and
> Web Entity JS message: https://s.ytimg.com/yts/jsbin/www-widgetapi-vfl8_u944/www-widgetapi.js 93 Error while parsing the 'allow' attribute: 'accelerometer;', 'autoplay;', 'encrypted-media;', 'gyroscope;', 'picture-in-picture' are invalid feature names.
> 
> [hifi-log.txt](https://github.com/highfidelity/hifi/files/2567934/hifi-log.txt)

These log messages are "OK" the are not cases of leaking asset URLs. Some of those error messages do look like they might be something that should be investigated in their own right, but they do not relate to this PR, and this PR should be considered passed if these are the only issues found.

**Comment by qlilaseguinolaza** [comment]
Currently testing

**Comment by qlilaseguinolaza** [comment]
@howard-stearns I'm seeing some url's for avatar fst and fbx files and some models in environments. Are these acceptable:

[11/12 12:20:47] [DEBUG] [hifi.networking.resource] Starting request for: http://mpassets.highfidelity.com/28569047-6f1a-4100-af67-8054ec397cc3-v1/LLMale2.fst

[11/12 12:20:47] [DEBUG] [hifi.networking.resource] Starting request for: http://mpassets.highfidelity.com/28569047-6f1a-4100-af67-8054ec397cc3-v1/LLMale2/LLMale2.fbx

[11/12 12:20:48] [DEBUG] [hifi.networking.resource] Starting request for: https://hifi-content.s3.amazonaws.com/luis/test_scripts/transitApp/animations/teleport01_warp.fbx

[11/12 12:26:57] [DEBUG] [hifi.networking.resource] Starting request for: http://headache.hungry.com/~seth/hifi/wow/wow-exports/world/wmo/azeroth/buildings/hearthglen/duskwoodbookshelf01.obj.gz

[11/12 12:27:02] [DEBUG] [hifi.networking.resource] Starting request for: http://mpassets.highfidelity.com/faf505f1-4fd1-4ed2-8909-816af246c48f-v1/M_VicGen/M_VicGen.fbx

[11/12 12:29:53] [DEBUG] [hifi.networking.resource] Starting request for: https://raw.githubusercontent.com/avatardz/reoiz/master/Vert_Avatar/Vert_Avatar/Vert_Avatar.fbx

[11/12 12:29:53] [DEBUG] [hifi.networking.resource] Starting request for: http://headache.hungry.com/~seth/hifi/soldier/Soldier_2k_poly/Soldier_2k_poly.fbx

[hifi-log.txt](https://github.com/highfidelity/hifi/files/2573352/hifi-log.txt)

**Comment by howard-stearns** [comment]
I don't imagine that it is acceptable - I would think the whole idea was to not show these (at the knowing expense of making our caching system even more undebuggable).

**Comment by qlilaseguinolaza** [comment]
Should I fail this for now or pause testing while you make more changes?

**Comment by howard-stearns** [comment]
let me look at it a bit

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15042/HighFidelity-Beta-PR14359-7a14cb1.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15043/HighFidelity-Beta-PR14359-7620335.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15043/HighFidelity-Beta-PR14359-7620335.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15043/HighFidelity-Beta-Interface-PR14359-7620335.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15043/HighFidelity-Beta-PR14359-7620335.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15043/HighFidelity-Beta-Interface-PR14359-7620335.exe

**Comment by qlilaseguinolaza** [comment]
Currently testing

**Comment by qlilaseguinolaza** [comment]
@howard-stearns Still seeing a few more model url's such as the following:

[11/13 08:35:53] [DEBUG] [hifi.modelformat] OBJ Reader material library "record-player.mtl" used in QUrl("http://headache.hungry.com/~seth/hifi/record-player/record-player.obj")

[11/13 08:35:53] [DEBUG] [hifi.modelformat] OBJ Reader material library "duskwoodbookshelf01.mtl" used in QUrl("http://headache.hungry.com/~seth/hifi/wow/wow-exports/world/wmo/azeroth/buildings/hearthglen/duskwoodbookshelf01.obj.gz")

[11/13 08:35:53] [DEBUG] [hifi.modelformat] OBJ Reader material library "inntablesmall.mtl" used in QUrl("http://headache.hungry.com/~seth/hifi/wow/wow-exports/world/wmo/azeroth/buildings/hearthglen/inntablesmall.obj.gz")

[11/13 08:35:53] [DEBUG] [hifi.modelformat] OBJ Reader material library "frame-1.mtl" used in QUrl("http://headache.hungry.com/~seth/hifi/quinn-gallery/picture-m/picture-m.obj")

[11/13 08:20:22] [DEBUG] [hifi.gpu.gl] variable / Strict texture  http://mpassets.highfidelity.com/52411f83-8043-46f4-8750-eee1bff221f5-v1/Entities/hextable2.fbx/tablenew.fbm

[11/13 08:44:07] [DEBUG] [hifi.gpu.gl] variable / Strict texture  https://s3.amazonaws.com/hifi-public/marketplace/hificontent/ozan/PalmTree.fbx/PalmTree.fbm/Palm1_Diffuse.tga

[11/13 08:44:08] [DEBUG] [hifi.gpu.gl] variable / Strict texture  https://s3.amazonaws.com/hifi-public/marketplace/hificontent/ozan/PalmTree.fbx/PalmTree.fbm/Palm2_Diffuse.tga

[11/13 08:44:08] [DEBUG] [hifi.gpu.gl] variable / Strict texture  https://hifi-public.s3.amazonaws.com/cozza13/apartment/kaya/Kaya.fbx/../../../../../mixamo/data/skins/kaya/kaya.fbm/OccupyGuy_diffuse.jpg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15082/HighFidelity-Beta-PR14359-938b45d.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15082/HighFidelity-Beta-PR14359-938b45d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15082/HighFidelity-Beta-Interface-PR14359-938b45d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15082/HighFidelity-Beta-PR14359-938b45d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15082/HighFidelity-Beta-Interface-PR14359-938b45d.exe

**Pull request merged**